### PR TITLE
Structure

### PR DESCRIPTION
**Pull request by ryanmalesic**
Initialize the basic structure

### CODE REVIEW

File: 2020sp-420-ProjectZero.iml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module external.linked.project.id="2020sp-420-ProjectZero" external.linked.project.path="$MODULE_DIR$" external.root.project.path="$MODULE_DIR$" external.system.id="GRADLE" external.system.module.group="" external.system.module.version="unspecified" type="WEB_MODULE" version="4" />
```
Comment (mslint) [inline 2020sp-420-ProjectZero.iml]: no newline at eof

File: build.gradle [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.5.2'
+    }
+
+    test {
+        useJUnitPlatform()
+        testLogging {
+            events "passed", "skipped", "failed"
+        }
+    }
+}
```
Comment (mslint) [inline build.gradle]: no newline at eof

File: cli-app/build.gradle [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+dependencies {
+}
+
+//This lets the shadow plugin know which is the main class
+jar {
+    manifest {
+        attributes 'Main-Class': 'projectzero.Main'
+    }
+}
```
Comment (mslint) [inline cli-app/build.gradle]: no newline at eof

File: core/src/test/java/projectzero/TestMain.java [hunk 4]
```diff
@@ -0,0 +1,12 @@
+package projectzero;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
```
Comment (student-dylan25) [inline core/src/test/java/projectzero/TestMain.java]: I'm not to familiar with Jupiter testing. Is this better than JUnit4 library? Just curious.

**Pull request merged**